dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
I assume this will change our bnodes globally, which is fine, but we need to make sure that these can still get labelled as anonymous in SciGraph.  We currently use the OwlAPI - http://owlapi.sourceforge.net/javadoc/org/semanticweb/owlapi/model/OWLIndividual.html#isAnonymous() method in particular -  to assign anonymous labels to nodes in Neo4J, and these labels are used elsewhere (cliqueLeader processing, generation of solr association views).

I'm not sure the best way to go about this, is there an explicit type "anonymous" in either the rdf or owl specs?  @cmungall @mbrush curious what your thoughts are here.